redaxo/src/core/lib/Fragment/Component/Alert/Error.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Redaxo\Core\Fragment\Component\Alert;
```

Comment:
@tbaddade arbeitet hier nun mit Namespaces. Um die Frage soll es jetzt noch nicht vordergründig gehen.
Müssen wir aber auch noch klären, wie wir damit in R6 umgehen (weiterhin gar nicht nutzen, oder Misch-Masch, oder komplett umstellen mit ggf. Aliasen für die alten Klassennamen).